docs/docs/usage/annotations.md

```diff
@@ -113,12 +113,12 @@ In order to use host modules, you need to import the relevant annotations, e.g.
 ```xml
 <dependency>
   <groupId>com.dylibso.chicory</groupId>
-  <artifactId>host-module-annotations-experimental</artifactId>
+  <artifactId>host-annotations</artifactId>
```

Comment:
Is the `host` part of the name here because this originally existed to hold host function annotations?  Or is it because your using them on the java host?  Do we ever see a different type of annotation than host annotations?  If not maybe we can simplify  the name.

Reply:
Historically it started as `host-module` as `@HostModule` was the only "main" annotation.
I kept `host` as, I think, it scopes a little the domain, i.e.: here there are annotations to be used for the Java Host

Not sure about the future, maybe we are going to need more annotations, for example, to control the execution of a module (think plain vs. virtual Threads etc.), and I don't know if we would use the same module.
Maybe I'm just overthinking and `annotations` is enough...

Reply:
Yeah given annotations are slim anyways... I think it would be ok have annotations for different use cases in one module.

Reply:
done :+1: